File: Csharp/SOMA_VETOR.cs

using System;
using System.Globalization;
internal class Program {
    private static void Main(string[] args) {
        CultureInfo CI = CultureInfo.InvariantCulture;
        int n, i;
        double soma, media;

        Console.Write("Quantos numeros voce vai digitar? ");
        n = int.Parse(Console.ReadLine());
        
        double[] vet = new double[n];
        for (i=0; i < n; i++) {
            Console.Write("Digite um numero: ");
            vet[i] = double.Parse(Console.ReadLine(), CI);
        }
        Console.WriteLine();
        Console.Write("VALORES = ");
        for (i = 0; i < n; i++) {
            Console.Write( vet[i].ToString("F1", CI) + " ");
        }
        Console.WriteLine();
        soma = 0;
        for (i = 0; i<n; i++) {
            soma = soma + vet[i];
        }
        Console.WriteLine("SOMA = " + soma.ToString("F2", CI));
 
        media = soma / n;
        
        Console.WriteLine("MEDIA = " + media.ToString("F2", CI));
    }
}

File: C/SOMA_VETOR.c
#include <stdio.h>

int main()
{
    int N, i;
    double soma, media;

    printf("Quantos numeros voce vai digitar? ");
    scanf("%d", &N);

    double vet[N];

    for (i = 0; i < N; i++){
        printf("Digite um numero: ");
        scanf("%lf", &vet[i]);
    }

    soma = 0;
    printf("\nVALORES = ");
    for (i = 0; i < N; i++){
        printf("%.1lf ", vet[i]);
    }
    printf("\n");

    for (i = 0; i < N; i++){
        soma = soma + vet[i];
        media = soma / N;
    }
    printf("SOMA = %.1lf\n", soma);
    printf("MEDIA = %.1lf\n", media);

    return 0;
}
